configure-hugepages/configure-hugepages.yaml

```diff
@@ -69,9 +69,9 @@ spec:
 
           function err() { echo >&2 "$@"; }
 
-          # Writes to the the nr_hugepages sysfs file for the given size (in
-          # kB), and then checks that we got that many hugepages afterward.
-          # Returns error if the result does not match.
+          # Writes to the nr_hugepages sysfs file for the given size (in kB),
```

Comment:
you can "squash" the commit so it doesn't show up as a separate commit in the history